source/MRViewer/MRViewer.cpp

```diff
@@ -704,6 +717,12 @@ Viewer::Viewer() :
     viewport_list.emplace_back();
     viewport_list.front().id = ViewportId{ 1 };
     presentViewportsMask_ |= viewport_list.front().id;
+
+#ifdef _WIN32
+    spaceMouseHandler_ = std::make_shared<SpaceMouseHandlerWindows>();
+#else
+    spaceMouseHandler_ = std::make_shared<SpaceMouseHandler>();
+#endif
```

Comment:
why have it as pointer, mb just store object? (looks like it is always initialized)

Reply:
Space MouseHandler have empty methods to avoid checking spaceMouseHandler_ for non-zero pointer
created separate initializer function